src/app/pages/property-details/confirmation-modal/confirmation-modal.component.spec.ts

```diff
@@ -0,0 +1,101 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { RouterTestingModule } from '@angular/router/testing';
+import { of, throwError } from 'rxjs';
+import {
+  resetSpies,
+  propertyDetailSpy,
+  toastServiceSpy,
+  routerSpy
+} from 'src/app/helpers/spies';
+import { ActivatedRoute, Router } from '@angular/router';
+import { NgxSpinnerModule } from 'ngx-spinner';
+import { ToastrService } from 'ngx-toastr';
+import { ConfirmationModalComponent } from './confirmation-modal.component';
```

Comment:
Hey @MBenedicte, we should use absolute imports here

Reply:
worked on that @CryceTruly